index2.html

```diff
@@ -0,0 +1,253 @@
+                ></path>
+              </g>
+            </svg>
+          </div>
+          <div class="p-2 mt-5">
+            <span class="header-font">Login</span>
+            <span class="rounded-top">Subscribe</span>
+          </div>
+        </div>
+
+        <!-- NAV BAR - menu row 1 -->
+        <nav class="navbar-expand-lg bg-light navbar-light">
+          <ul class="navbar-nav mx-auto d-flex flex-row">
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">U.S</a>
+            </li>
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">World</a>
+            </li>
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">Business</a>
+            </li>
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">Tech & Science</a>
+            </li>
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">Culture</a>
+            </li>
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">Newsgeek</a>
+            </li>
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">Sports</a>
+            </li>
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">Health</a>
+            </li>
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">The Debate</a>
+            </li>
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">Experts</a>
+            </li>
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">Vantage</a>
+            </li>
+            <li class="nav-item flex-grow-1 border-right p-1">
+              <a class="nav-link" href="#">Search</a>
```

Comment:
- [ ] Search should use a form element with a search input.

Reply:
Done